provider/datagen/tests/data/json/percent/essentials@1/und.json

```diff
@@ -0,0 +1,10 @@
+{
+  "number_index": 0,
+  "percent_sign_affixes": {
+    "prefix": "",
+    "suffix": ""
+  },
+  "percent_sign_symbol": "%",
+  "percent_symbol_index": 5,
+  "standard": "#,##0%"
```

Comment:
is `standard` fully determined by the locales number format, and the other fields?

Reply:
`standard` is just a pass through from the actual object
```
"percentFormats-numberSystem-latn": {
    "standard": "#,##0%"
},
```